src/pages/editor/PublicationChannelTableRow.tsx

```diff
@@ -0,0 +1,37 @@
+import { Chip, TableCell, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { useFetchPublisher } from '../../api/hooks/useFetchPublisher';
+import { ClaimedChannel } from '../../types/customerInstitution.types';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { getLanguageString } from '../../utils/translation-helpers';
+
+interface ChannelClaimTableRowProps {
+  channel: ClaimedChannel;
+}
+export const ChannelClaimTableRow = ({ channel: claim }: ChannelClaimTableRowProps) => {
+  const { t } = useTranslation();
+  const publisher = useFetchPublisher(getIdentifierFromId(claim.channelClaim.channel));
+  const publisherName = publisher ? publisher.data?.name : '';
+
+  const organization = useFetchOrganization(claim.claimedBy.organizationId);
+  const organizationName = organization ? getLanguageString(organization.data?.labels) : '';
+
+  return (
+    <TableRow key={claim.id}>
+      <TableCell sx={{ verticalAlign: 'top' }}>
+        {publisherName ? publisherName : <span style={{ fontStyle: 'italic' }}>{t('common.unknown')}</span>}
+      </TableCell>
+      <TableCell sx={{ verticalAlign: 'top' }}>
+        {organizationName ? organizationName : <span style={{ fontStyle: 'italic' }}>{t('common.unknown')}</span>}
+      </TableCell>
+      <TableCell sx={{ verticalAlign: 'top' }}>{claim.channelClaim.constraint.publishingPolicy}</TableCell>
+      <TableCell sx={{ verticalAlign: 'top' }}>{claim.channelClaim.constraint.editingPolicy}</TableCell>
+      <TableCell sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', verticalAlign: 'top' }}>
+        {claim.channelClaim.constraint.scope.map((scope, index) => (
+          <Chip key={index} label={scope} />
```

Comment:
`scope` burde være unikt for hver claim
```suggestion
        {claim.channelClaim.constraint.scope.map((scope) => (
          <Chip key={scope} label={scope} />
```